config/patches/libxml2/libxml2-mingw.patch

```diff
@@ -0,0 +1,227 @@
+--- libxml2-2.9.0/uri.c.orig	2013-03-31 04:05:23 +0400
```

Comment:
For the patches here, @scotthain suggested we thank Alex Pawlow (https://github.com/Alexpux) and other msys2 contributors from https://github.com/Alexpux/MINGW-packages